src/hooks/useDarkMode.js

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from "react";
+import { useLocalStorage } from "./useLocalStorage";
+
+function useToggle(darkMode, target) {
+  useEffect(() => {
+    console.log("dark mode = ", darkMode);
+    console.log("local storage = ", window.localStorage.getItem("dark"));
+    let darkTarget = document.querySelector(target);
+    if (darkMode === true) {
+      darkTarget.classList.add("dark-mode");
+    } else if (darkMode === false) {
+      darkTarget.classList.remove("dark-mode");
+    } else {
+      console.log("something went wrong");
+    }
```

Comment:
I think this was the solution to the issue we were having yesterday. For some reason, having a .toggle function instead of .add and .remove was causing the application to break.